src/app/lib/utilities/getUrlHelpers/getEmbedUrl/index.test.js

```diff
@@ -45,18 +33,6 @@ const testCases = [
       queryString: testOverrideParam,
     },
   },
-  {
-    description: `should build an AMP url for articles in test environment with test override`,
-    expected: `https://polling.test.bbc.co.uk/ws/av-embeds/articles/${mediaId}/amp`,
-    environment: 'test',
-    before: setEnvironment,
-    embedObject: {
-      isAmp: true,
-      mediaId,
-      type: 'articles',
-      queryString: testOverrideParam,
-    },
-  },
   {
     description: `should build a CANONICAL url for articles in test environment with live override`,
     expected: `/ws/av-embeds/articles/${mediaId}${embedUrlLiveOverride}`,
```

Comment:
Does the Canonical reference need to stay in these tests or should I remove it for the remaining tests now that there's no longer AMP references anywhere in these files?

Reply:
(referring to line 62...not the ones highlighted in red)